app/helpers/helpers.rb

```diff
@@ -0,0 +1,5 @@
+def helper_category(category)
```

Comment:
This is a weird helper name and I'm not sure what its doing. Can you come up with a better name or delete this helper?